.env.template

```diff
@@ -1,3 +1,3 @@
 TWILIO_ACCOUNT_SID=ACxxxxxxxxxx
-TWILIO_API_KEY=SKxxxxxxxxxx
-TWILIO_API_SECRET=xxxxxxxxxx
+TWILIO_API_KEY_SID=SKxxxxxxxxxx
+TWILIO_API_KEY_SECRET=xxxxxxxxxx
```

Comment:
@PikaJoyce On second thought, I don't think we should change this because existing users might get affected by this when they pull in the latest changes from `master`. Suddenly their app will no longer work.

Reply:
Would this be the case even if I've changed not only the template but all the variables in the files that use them as well?

Reply:
I think this change should still be made in the Ahoy app. There are no existing users that would be affected. 

https://github.com/twilio/twilio-video-app-react/pull/75

This change would mean that the App and the Quickstart would have different names for their environment variables. Think that is ok?

Reply:
I think it's okay. But maybe need thoughts from @manjeshbhargav. I'm not sure if it's more important to keep it consistent across all platforms, and simply notifying folks of this change and having everyone change their personal .env file OR just changing the read me so it makes more sense intuitively?

Reply:
@PikaJoyce @timmydoza I think the differences in names are fine, since we don't want to break existing QuickStart users, and the Ahoy App is new. However, the description for how to create these strings should be the same.